_includes/sidebar.html

```diff
@@ -1,9 +1,4 @@
-<div class="col col-4 sm-col-14 pt-2">
-  <p class="f4 sm-f2 mb-3">
-    <a class="button sm-button-full" href="{{ site.consultation }}">
-      Book a Consultation
-    </a>
-  </p>
+  <div class="col col-4 sm-col-14 pt-2">
```

Comment:
```suggestion
<div class="col col-4 sm-col-14 pt-2">
```

Reply:
Just getting rid of the extra space. It would still work but having the spacing makes it easier to see which boys are inside of who.